common-lib/src/cmr/common/api/errors.clj

```diff
@@ -27,8 +27,8 @@
    :body {:errors ["An Internal Error has occurred."]}})
 
 (defn- keyword-path->string-path
-  "Converts a set of keyword field paths into the string equivalent field paths to return to the
-  user."
+  "Converts a set of keyword field paths into the string equivalent field paths
```

Comment:
Oh, btw, I would expect that kibit and eastwood is what we'd use in CI/CD scripts; I find that bikeshed and yagni are great for catching the odd bits that might get missed elsewhere, but are too noisy and often go above/beyond community norms for every-day linting/static analysis ... great as a developer tool, maybe a bit much as part of automation?